daemons/ipa-kdb/tests/ipa_kdb_tests.c

```diff
@@ -478,6 +494,44 @@ void test_dom_sid_string(void **state)
 }
 
 
+void test_check_trusted_realms(void **state)
+{
+    struct test_ctx *test_ctx;
+
+    test_ctx = (struct test_ctx *) *state;
+
+    for(size_t i = 0; i < NUM_SUFFIXES; i++) {
+        char *test_realm = NULL;
+        char *trusted_realm = NULL;
+        asprintf(&test_realm, TEST_REALM_TEMPLATE, i);
+	krb5_error_code kerr = 0;
+
+        if (test_realm) {
```

Comment:
I think this implies that if there is low memory and the asprintf fails (test_realm==NULL) then the test won't be executed.

Reply:
Just to keep a good hygiene here.